pandas/tests/io/test_gcs.py

```diff
@@ -56,7 +56,15 @@ def open(*args):
 
     monkeypatch.setattr("gcsfs.GCSFileSystem", MockGCSFileSystem)
     df1.to_csv("gs://test/test.csv", index=True)
-    df2 = read_csv(StringIO(s.getvalue()), parse_dates=["dt"], index_col=0)
```

Comment:
This no longer works since file needs to be closed to validate credentials. https://github.com/pandas-dev/pandas/issues/32486#issuecomment-595992013